pkg/database/workouts_map.go

```diff
@@ -181,17 +176,6 @@ func createMapData(gpxContent *gpx.GPX) *MapData {
 				uphill += segment.UphillDownhill().Uphill
 				downhill += segment.UphillDownhill().Downhill
 				maxSpeed = max(maxSpeed, segment.MovingData().MaxSpeed)
-
-				firstPoint := &segment.Points[0]
-				if lastSegmentEnd != nil {
-					// Calculate time between this segment's first point and last segment's last point, and add to pause time
-					pauseDuration += firstPoint.Timestamp.Sub(lastSegmentEnd.Timestamp)
```

Comment:
Is this not needed either? I was only talking about the `pauseDistance`, not the `pauseDuration`, in my review...

Reply:
Yes, we should add the pause duration between segments to the pause duration, but I am not totally sure all the existing calculations will then be correct?

Reply:
OK, so I just checked the stats for my run this morning, which has a 5 min pause between segments. It is a bit difficult comparing against e.g. Strava because that does some auto-pausing stuff as well, but anyways... The "average tempo" now appears to show the tempo *while moving* (so: `totalDistance / totalTime` for me) and the "average tempo without pause" now shows a tempo that is way too fast (it appears to do `totalDistance / (totalTime - pauseDuration)` which is not right as in my case the pause duration happens between segments and hence is outside `totalTime`).

In the end I think we should calculate as follows:
* Average tempo: totalDistance / (last_track.last_segment.last_point.time - first_track.first_segment.first_point.time). Just the distance ran divided by the time it took. Perhaps the distance moved while paused between segments should also be included in this (but we can then only assume that was a straight line)
* Average tempo without pause: totalDistance / totalTime (this means we interpret `totalTime` as 'time moving' excluding any pauses. So then also subtract the pause duration *within* a segment from totalTime)

Reply:
Is it better/easier if we remove the pause durations completely for now? Then you can try some cases until you find a good calculations...

Reply:
> Is it better/easier if we remove the pause durations completely for now? Then you can try some cases until you find a good calculations...

Yes, that seems reasonable (note that the map still draws a contiguous line for pauses which may suggest that gaps between segments are still counted... should figure out if we can draw these gaps as actual gaps)

Reply:
Yes, was also on my wishlist to have some markers when you paused, so you find them easier... (even if you didn't move during that pause, eg. at a traffic light)